wiki/Article_Styling_Criteria/en.md

```diff
@@ -488,13 +502,25 @@ There are no visual differences between external and osu! web links. Due to this
 
 ### Images
 
-Like the [links](#links), use either the reference or inline style. Use either appropriately.
+Images must use the inline style if they are only referenced once; otherwise, it may use the reference style. References to reference-styled links must be placed at the bottom of the article.
```

Comment:
we don't follow this in practice

either change "must" to "should" or detail the cases where you might use reference regardless of this rule (images in line with text with long URIs)